docs/pages/release_notes.md

```diff
@@ -24,6 +24,27 @@ This is a {{ site.pmd.release_type }} release.
 
 ### 🚀 New and noteworthy
 
+#### New Rule UnnecessaryWarningSuppression (experimental)
+
+This new Java rule {% rule java/bestpractices/UnnecessaryWarningSuppression %} reports unused suppression
+annotations and comments. The rule class supports any PMD language,
```

Comment:
I don't think we should mention that the rule class supports all languages. The java rule supports only java, and this section is about the java rule. The actual rule class in pmd-core is not mentioned here. It is also an implementation detail that it supports any language.